src/get-started/java-binding.md

```diff
@@ -222,6 +222,26 @@ reference. Light clients also provide access to information on the
 [transaction][exonum-transaction] execution result
 (which may be either success or failure) to their users.
 
+### Blockchain Events
+
+A service can also handle a block commit event that occurs each time
+the framework commits a new block. The framework delivers this event to
+implementations of [`Service#afterCommit(BlockCommittedEvent)`][service-after-commit]
+in each deployed service. Each node in the network processes that event
+independently from other nodes. The event includes a `Snapshot`,
+allowing a read-only access to the database state _exactly_ after the commit
+of the corresponding block.
+
+As services can read the database state in the handler, they may detect
+any changes in it, e.g., that a certain transaction is executed;
+or some condition is met. Services may also create and submit new transactions
+using [`Node#submitTransaction`][node-submittransaction]. Using this callback
```

Comment:
Nit: why do we use `submittransaction` here, but `after-commit` above? (I like hyphens better)